system_test.sh

```diff
@@ -37,6 +49,20 @@ clear_tmp='rm -rf $tmp'
 trap "$clear_tty; $clear_tmux; $clear_tmp" EXIT
 trap 'printf "\nTerminated by SIGINT at line $LINENO\n"; exit 1' INT
 
+# required binary
+
+# This is specifically needed inside the system_test.sh to ensure it uses the
+# built entr binary and not a pre-existing system binary
+ENTR="$(./entr 2>&1)"
```

Comment:
How does this alter the behavior of the tests? All tests use `./entr`, so it a pre-existing binary will never be used

Reply:
I remember this was needed when I added it but I can't recall the details now, so I will just drop this section for now.